tools/attrd_updater.c

```diff
@@ -431,6 +431,7 @@ static int
 send_attrd_query(pcmk__output_t *out, const char *attr_name,
                  const char *attr_node, gboolean query_all)
 {
+    uint32_t options = 0;
```

Comment:
copyright in this file

Reply:
I think you meant in lib/common/ipc_attrd.c.  Fixed.

Reply:
Whoops, I scrolled too far down.